src/script.js

```diff
@@ -57,9 +102,34 @@ function validaDescrizione(desc) {
     return true;
 }
 
+function validaNome(username) {
+    const regex = /^[a-zA-Z0-9]+$/;
```

Comment:
```suggestion
    const regex = /[\W]/;
```
La regex non è uguale a quella del .php

Reply:
Ci avevo provato ma non va, il comportamento delle regex in JS e PHP è diverso in certi casi